mmv1/third_party/terraform/go.mod.erb

```diff
@@ -26,9 +26,9 @@ require (
 	github.com/sirupsen/logrus v1.8.1
 	golang.org/x/net v0.20.0
 	golang.org/x/oauth2 v0.16.0
-	google.golang.org/api v0.156.0
```

Comment:
Do you need these version bumps? It seems like they should not be needed considering this is already released under the beta version. If we don't need them, they should probably be removed from the PR.

Reply:
I definitely need them. I had the code ready a long time ago, but the tests weren't passing, as it didn't build.

wrt. Tests - I did run them, they pass with no issues

Reply:
Ok, so just to clarify, you're saying this behavior is not working right now in beta, and the dependency bumps will fix that for beta (and now GA as well)?